tests/vs/tests.vcxproj

```diff
@@ -185,6 +185,12 @@
     <VcpkgEnableManifest>true</VcpkgEnableManifest>
     <VcpkgAutoLink>false</VcpkgAutoLink>
   </PropertyGroup>
+  <PropertyGroup Label="Vcpkg" Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">
+    <VcpkgTriplet>arm64-windows</VcpkgTriplet>
+  </PropertyGroup>
```

Comment:
For the debug `PropertyGroup`, might also need:
```xml
<VcpkgConfiguration>Debug</VcpkgConfiguration>
```

Reply:
`<VcpkgConfiguration>` isn't set anywhere in the test project. It is set in the GUI though, so I dunno what's going on there. I don't think I'll make any changes here.

Reply:
I just had noticed that in all of the **debug** property groups (across all the project files), that we explicitly specify the **debug** vcpkg configuration.

On the other hand, for the **release** property groups (across all files), we don't include the vcpkg configuration (perhaps because **release** is default?).

So this comment here just catches the only place where we're not specifying the debug configuration inside the debug property group. (So.. we're probably getting release libs; I dunno if we case though! Maybe not a big deal).

Reply:
I dunno what's going on, as I said, it's set in the GUI.

![image](https://github.com/user-attachments/assets/5e64728b-3b4a-48c4-a56f-6cfef27bf17b)

Reply:
Ahh; thanks - I didn't make that connection!
I guess that setting is just somewhere else at a higher level in the XML 😌